test/TestCompoundMath.sol

```diff
@@ -1,93 +1,30 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.0;
 
+import {CompoundMathMock} from "./mocks/CompoundMathMock.sol";
+import {CompoundMathRef} from "./references/CompoundMathRef.sol";
 import "forge-std/Test.sol";
-import "src/math/CompoundMath.sol";
-import "./references/CompoundMathRef.sol";
-
-contract CompoundMathFunctions {
-    function mul(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMath.mul(x, y);
-    }
-
-    function div(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMath.div(x, y);
-    }
-}
-
-contract CompoundMathFunctionsRef {
-    function mul(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMathRef.mul(x, y);
-    }
-
-    function div(uint256 x, uint256 y) public pure returns (uint256) {
-        return CompoundMathRef.div(x, y);
-    }
-}
 
 contract TestCompoundMath is Test {
-    uint256 internal constant SCALE = 1e36;
-    uint256 internal constant WAD = 1e18;
-
-    CompoundMathFunctions compoundMath;
-    CompoundMathFunctionsRef compoundMathRef;
+    CompoundMathMock mock;
+    CompoundMathRef ref;
 
     function setUp() public {
-        compoundMath = new CompoundMathFunctions();
-        compoundMathRef = new CompoundMathFunctionsRef();
+        mock = new CompoundMathMock();
+        ref = new CompoundMathRef();
     }
 
     /// TESTS ///
 
     function testMul(uint256 x, uint256 y) public {
-        unchecked {
-            vm.assume(y == 0 || (x * y) / y == x);
-        }
-
-        assertEq(CompoundMath.mul(x, y), CompoundMathRef.mul(x, y));
-    }
-
-    function testMulOverflow(uint256 x, uint256 y) public {
```

Comment:
Is it delete because it's tested with certora?

Reply:
But why `testMul` is kept then?

Btw one I thought about after is that external contributors may not be able to create certora tests and without testing example it might be difficult for them to add tests

Reply:
1. For gas tracking: it allows to include the gas consumption into the gas report

2. Yes, and the argument also holds for the team: by introducing this change, we expect from contributors to develop & run tests using Certora only

I'm ok with that, as any open-source project relies on a specific tech stack which may not be easily mastered... But I don't see this as a viable argument against moving to Certora tests in this case, as we can safely state that Certora > foundry fuzzing

Also consider the origin of this PR: in order to add a feature to the repo, I had to write the exact same tests twice (once with Certora, once with forge)

Reply:
1. For gas tracking ok but you're removing the gas comparison with the reference library right? I'm against it because one of the goals is to be sure that our lib is at least better than the reference one.

2. Yes but with Certora you need a key to run tests if I'm correct, while you don't need it for forge tests. So there are more constraints to writing certora tests than pure forge tests. Not a blocker to me, we can keep the current version but I think we must keep that in mind.

Reply:
> you're removing the gas comparison with the reference library right?

As you can see, I compare the result of our own library vs the result of the reference library. So the gas consumption of both libraries are tracked. Am I missing something?

> with Certora you need a key to run tests

This is something I wasn't aware of. We should discuss this then

Reply:
Ok I get it now. It's just that it seems to be a test and not a gas comparison check when reading it. So it's like if we were testing only this the correct behavior for this condition: `vm.assume(y == 0 || x < type(uint256).max / y);`. 

I would add Gas in the naming and remove: `/// TESTS ///` because it's misleading, no? (perhaps only on my side).

How did you run certora tests locally?

Reply:
I think that we should keep both ! (We only remove testGas..)